src/utils/Makefile

```diff
@@ -0,0 +1,2 @@
+SRC += $(SRCDIR)utils/common_utils.c
+SRC += $(SRCDIR)utils/logger.c
```

Comment:
No EOL (i.e. end of line character) at the end of file.

Besides github being annoying about it, everyone else's editor makes sure to put EOL at the end of file, so leaving it would inevitably lead to an unneeded line changed in git logs.